automation/jenkins_build.sh

```diff
@@ -218,22 +218,29 @@ if [ "$deploy" == "yes" ]; then
 		-e SLUG="$SLUG" \
 		-e BUILD_VERSION="$S3_VERSION_HOSTOS" \
 		-e DEVELOPMENT_IMAGE="$DEVELOPMENT_IMAGE" \
+		-e DEPLOYER_UID=$(id -u) \
```

Comment:
You stopped reading one sentence too soon @agherzan, the very next sentence is:
> When passing a numeric ID, the user does not have to exist in the container.

:P